src/main/java/com/workshop/demo/service/impl/RestaurantServiceImpl.java

```diff
@@ -0,0 +1,74 @@
+package com.workshop.demo.service.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import com.workshop.demo.exception.ResourceNotFoundException;
+import com.workshop.demo.exception.BlogapiException;
+import com.workshop.demo.model.Restaurant;
+import com.workshop.demo.model.User;
+import com.workshop.demo.payload.ApiResponse;
+import com.workshop.demo.payload.RestaurantRequest;
+import com.workshop.demo.payload.RestaurantResponse;
+import com.workshop.demo.repository.RestaurantRepository;
+import com.workshop.demo.service.RestaurantService;
+
+@Service
+public class RestaurantServiceImpl implements RestaurantService {
+    @Autowired
+    private RestaurantRepository restaurantRepository;
+
+    private static final String ID_STR = "id";
+
+    private static final String THIS_RESTAURANT = " this restaurant";
+
+    // return all the restaurants' names in our blog
+    @Override
+    public RestaurantResponse<Restaurant> getAllRestaurants() {
+
+        return null;
+    }
+
+    // return the score of one specific restaurant with restaurantRequest
+    @Override
+    public Integer getScore(RestaurantRequest restaurantRequest) {
```

Comment:
Should we get the score by getting the average score of the restaurant from Reviews?